main/1-Inf.rev (Anual)/SQL_1.sql

```diff
@@ -0,0 +1,226 @@
+SELECT
+      DISTINCT  D.CUIIO,
+       R.IDNO,
+       R.DENUMIRE,
+       C4.CODUL4,
+       R.CUATM,
+       R.CAEM2,
+       C.FULL_CODE,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('110') THEN  D.COL1 ELSE NULL END ) AS  R_110_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('111') THEN  D.COL1 ELSE NULL END ) AS  R_111_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('112') THEN  D.COL1 ELSE NULL END ) AS  R_112_COL1, 
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('113') THEN  D.COL1 ELSE NULL END ) AS  R_113_COL1,  
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('120') THEN  D.COL1 ELSE NULL END ) AS  R_120_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('130') THEN  D.COL1 ELSE NULL END ) AS  R_130_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('130') THEN  D.COL2 ELSE NULL END ) AS  R_130_COL2,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('140') THEN  D.COL1 ELSE NULL END ) AS  R_140_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('140') THEN  D.COL2 ELSE NULL END ) AS  R_140_COL2,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('141') THEN  D.COL1 ELSE NULL END ) AS  R_141_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('150') THEN  D.COL1 ELSE NULL END ) AS  R_150_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('150') THEN  D.COL2 ELSE NULL END ) AS  R_150_COL2,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('161') THEN  D.COL1 ELSE NULL END ) AS  R_161_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('162') THEN  D.COL1 ELSE NULL END ) AS  R_162_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('163') THEN  D.COL1 ELSE NULL END ) AS  R_163_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('164') THEN  D.COL1 ELSE NULL END ) AS  R_164_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('165') THEN  D.COL1 ELSE NULL END ) AS  R_165_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('170') THEN  D.COL1 ELSE NULL END ) AS  R_170_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('170') THEN  D.COL2 ELSE NULL END ) AS  R_170_COL2,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('180') THEN  D.COL1 ELSE NULL END ) AS  R_180_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('180') THEN  D.COL2 ELSE NULL END ) AS  R_180_COL2,
+       SUM(CASE WHEN D.CAPITOL = 1181 AND  D.RIND IN ('181') THEN  D.COL1 ELSE NULL END ) AS  R_181_COL1,    
+       SUM(CASE WHEN D.CAPITOL = 1182 AND  D.RIND IN ('210') THEN  D.COL1 ELSE NULL END ) AS  R_210_COL1,
+       SUM(CASE WHEN D.CAPITOL = 1182 AND  D.RIND IN ('210') THEN  D.COL2 ELSE NULL END ) AS  R_210_COL2,
```

Comment:
SUM(CASE WHEN D.CAPITOL = 1182 AND  D.RIND IN ('210') THEN  D.COL2 ELSE NULL END ) AS  R_210_COL2

Reply:
SUM(CASE WHEN D.CAPITOL = 1182 AND  D.RIND IN ('210') THEN  D.COL2 ELSE NULL END ) AS  R_210_COL2